.travis.yml

```diff
@@ -26,6 +26,22 @@ matrix:
               packages:
                 - g++-6
           env: CXX_COMPILER=g++-6 BUILD_TYPE=Release COVERAGE=OFF
+        - os: linux
+          addons:
+            apt:
+              sources:
+                - ubuntu-toolchain-r-test
+              packages:
+                - g++-7
+          env: CXX_COMPILER=g++-7 BUILD_TYPE=Release COVERAGE=OFF CXX17_TESTS=ON
```

Comment:
`CXX17_TESTS=ON` is no longer needed, if I understand correctly.

Reply:
Yes indeed, it should have been removed.